langfuse-core/src/types.ts

```diff
@@ -2,6 +2,10 @@ import { type LangfusePromptClient } from "./prompts/promptClients";
 import { type components, type paths } from "./openapi/server";
 
 export type LangfuseCoreOptions = {
+  // Langfuse API publicKey obtained from the Langfuse UI project settings
+  publicKey?: string;
+  // Langfuse API secretKey obtained from the Langfuse UI project settings
+  secretKey?: string;
```

Comment:
adding the keys here does not work unfortunately as we only expect publickey for LangfuseWebStateless

Reply:
Ok, i see that it technically works, but a user of the web sdk would see the secret key as an option in their intellisense. I think we should not show that to our users as this could trigger setting secret keys into the browser. Maybe we can omit the secret key for the web options?